extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderLightbox.php

```diff
@@ -81,36 +82,46 @@
 					</select>
 				</div>
 				<div class="input-group">
-					<label for="occupation"><?= wfMsg('user-identity-box-about-occupation'); ?></label>
-					<input type="text" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>" />
+					<label for="occupation"><?= wfMessage( 'user-identity-box-about-occupation' )->escaped(); ?></label>
+					<input type="text" id="occupation" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>"/>
+				</div>
+				<div class="input-group">
+					<label for="gender"><?= wfMessage( 'user-identity-box-about-gender' )->escaped(); ?></label>
+					<input type="text" id="gender" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>"/>
 				</div>
 				<div class="input-group">
-					<label for="gender"><?= wfMsg('user-identity-box-about-gender'); ?></label>
-					<input type="text" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>" />
+					<label for="website"><?= wfMessage( 'user-identity-box-about-website' )->escaped(); ?></label>
+					<input type="text" id="website" name="website" value="<?= $user['website']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="website"><?= wfMsg('user-identity-box-about-website'); ?></label>
-					<input type="text" name="website" value="<?= $user['website']; ?>">
+					<label for="twitter"><?= wfMessage( 'user-identity-box-about-tweet' )->escaped(); ?></label>
+					<label for="twitter" class="tweet-at">twitter.com/</label>
+					<input type="text" id="twitter" name="twitter" value="<?= $user['twitter']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="twitter"><?= wfMsg('user-identity-box-about-tweet'); ?></label>
-					<span class="tweet-at">@</span>
-					<input type="text" name="twitter" value="<?= $user['twitter']; ?>">
+					<label for="fbPage"><?= wfMessage( 'user-identity-box-about-fb-page' )->escaped(); ?></label>
+					<label for="fbPage" class="fb-start">facebook.com/</label>
+					<input type="text" id="fbPage" name="fbPage" value="<?= $user['fbPage']; ?>">
```

Comment:
Just for reference, the single quotes will still display fine to the user, they'll just be HTML encoded (as `&#039;`) in the source, so there's no problem doing that for all this data if that simplifies things. If you're escaping the content of attributes, you can make use of the helper method [`Sanitizer::encodeAttribute`](https://github.com/Wikia/app/blob/5690ef15aa2/includes/Sanitizer.php#L1120-L1133) which does this and cleans up whitespace a bit. Also, `wfMessage()->escaped()` always [escapes single quotes](https://github.com/Wikia/app/blob/5690ef15aa2/includes/Message.php#L405), so you don't have to worry if you see that in an attribute either. :)